pkg/network/server.go

```diff
@@ -976,8 +963,8 @@ func (s *Server) handleTxCmd(tx *transaction.Transaction) error {
 	s.txInMap[tx.Hash()] = struct{}{}
 	s.txInLock.Unlock()
 	if s.verifyAndPoolTX(tx) == nil {
-		if s.consensus != nil {
-			s.consensus.OnTransaction(tx)
+		if s.txCallback != nil {
```

Comment:
BTW, now mempool has its own subscriptions mechanism, so maybe use it and remove `txCallback` from the server?

Reply:
Haven't thought about it, let me take a look.

Reply:
Seems to me we can't really do that and we actually need to fix current tx handling logic wrt old #668 issue. The problem is that consensus should get transactions irrespective of whether we add them into the main mempool or not.

Reply:
Yes, seems that this problem with conflicting transactions is still unresolved and we may reopen this issue. And it seems to me that this problem is not only about transactions that have `Conflicts` attribute, but about any other conflicting transactions, e.g. when sender has funds to pay for transaction X *or* for transaction Y, but not for both.

Reply:
I've fixed it, pretty trivial.